webhook_base/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Hoang Tran <thhoang.tr@gmail.com>.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Webhook Base",
+    "summary": "Webhook to publish events based on automated triggers",
+    "version": "16.0.0.0.1",
+    "author": "Hoang Tran,Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "website": "https://github.com/OCA/webhook",
+    "depends": [
+        "base",
```

Comment:
We don't need this dependency because "base_automation" already depends on "base".